modules/market/src/main/java/com/opengamma/strata/market/curve/CurveGroup.java

```diff
@@ -80,7 +80,10 @@ public static CurveGroup of(CurveGroupName name, Map<Currency, Curve> discountCu
   /**
    * Creates a curve group using a curve group definition and some existing curves.
    * <p>
-   * The curves must include all curves named in the curve group definition.
+   * If there are curves named in the definition which are not present in the curves the group is built using
+   * whatever curves are available.
+   * <p>
+   * If there are multiple curves with the same name in the curves it is undefined which one is used.
```

Comment:
```
   * If there are curves named in the definition that are not present in the input curves,
   * the group is built using whatever curves are available.
   * <p>
   * If there are multiple curves with the same name in the set of input curves, it is undefined which one is used.
```

I can't say I like the "undefined" part of this, but unless we pass a `Map<CurveName, Curve>` in (and validate it) things could get messy.